### DISCUSSION TITLE
Unable to Establish Connection Between Base Agents After Adding Tenants Module

### DISCUSSION THREAD

**Discussion by tarunvadde4** [openwallet-foundation/credo-ts#1503] [Q&A]
I am encountering an issue when attempting to establish a connection between two different base agents after integrating the Tenants module in my implementation of the AFJ (Aries Framework JavaScript) version 0.4.0. This issue arises while initializing an agent using the provided configuration.

```
const agent1 = new Agent({
  config: agent1Config,
  modules: {
    tenants: new TenantsModule(),
    indySdk: new IndySdkModule({ indySdk }),
    connections: new ConnectionsModule({
      autoAcceptConnections: true,
    }),
  },
  dependencies: agentDependencies,
});
```

`Error: Couldn't determine tenant id for inbound message. Unable to create context.
`

**Comment by TimoGlastra** [comment]
Closing as it's based on an older version of Credo. Please open a new issue if you're still having problems. Thanks!
